nesta/packages/mag/query_mag_api.py

```diff
@@ -209,65 +214,53 @@ def update_field_of_study_ids(mag_subscription_key, session, fos_ids):
     logging.info("Added new fields of study to database")
 
 
-if __name__ == "__main__":
```

Comment:
I removed this since it was dependent on Russ's directory structure, and wasn't necessary for testing anyway